src/data.toml

```diff
@@ -1,22 +1,21 @@
-addon_404 = "You are not supposed to visit the url directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
-not_support_channel = "This is not the support channel, please move to https://discord.com/channels/1196075698301968455/1210652540757090315."
-cloudsync_not_working = "You might be using an outdated version of Cloud Sync, as older versions don't work anymore.\n1. Make sure you have the latest version of Cloud Sync (available in the pins) available\n2. If that doesn't work, and you have **antied** installed, uninstall and reinstall it (https://angelix1.github.io/MP/angel/antied/)\n### (It is advised to use local backup to avoid accidental erasure of cloud data as it can't be recovered once overridden with a empty sync.)"
+ios_app_icons = "When sideloading with a paid certificate the application will be signed using a wildcard App ID.\nThis effectively breaks being able to change app icons.\nYou need to sign the application with a local dev CERT instead if you want this functionality.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
+addon_404 = "You are not supposed to visit the URL directly since it does not serve anything that can be rendered. Copy the link and use it to install the add-on."
+cloudsync_not_working = "You might be using an outdated version of Cloud Sync, as older versions don't work anymore.\n It is recommended to use local backup to avoid accidental erasure of cloud data as it can't be recovered once overridden with a empty sync.\n1. Make sure you have the latest version of Cloud Sync (available in the pins)\n2. Uninstall and reinstall it (https://angelix1.github.io/MP/angel/antied/) if it's installed."
 android15_error = "Refer to https://discord.com/channels/1196075698301968455/1205101050739167272/1264529990997905442"
-fakeprofilesthemeandeffects_guide = "## How to use Fakeprofilesthemeandeffects plugin-\n\n1 - Select colour\n2 - Select colour\n3 - Select effect\n4 - Copy FPTE\n5 - Paste what you just copied into about me, on a new line\n-# Note: you *might* experience using gboard clipboard when pasting the PTE. Instead use hold to paste.\n [󠀀](https://imgur.com/paPTG3J)"
+fakeprofilesthemeandeffects_guide = "## How to use Fakeprofilesthemeandeffects plugin-\n\n1 - Select your preferred colours & effects in 'Primary' & 'Accent' & 'Effect' fields respectively.\n2 - Copy FPTE through the button provided in the right side of FPTE Builder\n3 - Paste the copied FPTE into about me section under your profile in a new line.\n-# Note: you *might* experience issues on not correctly applying effects or profile colours using gboard clipboard when pasting the FPTE characters as gboard doesn't support some FPTE characters and removes them while pasting from the keyboard, long press to paste the FPTE instead to get around this issue.\n [󠀀](https://imgur.com/paPTG3J)"
```

Comment:
besides the triple ampersand here, most of these phrases still really just are grammatical sauce without meatballs, I'd suggest you run these through Grammarly and let them be rephrased in a concise manner

Reply:
changed a bit. review again if needed.